### PR TITLE
Slight adjustments to lobby example. Removing redundancies. First pr

### PR DESCRIPTION
**Pull request by Sirmadeira**

### CODE REVIEW

File: examples/lobby/src/server.rs [hunk 1]
```diff
@@ -241,11 +237,12 @@ mod lobby {
...
                     },
                 );
-            } else {
+            } // Common scenario player just started and everyone is in lobby
+            else {
                 if host.is_none() {
                     // one of the players asked for the game to start
                     for player in &lobby.players {
-                        error!("Spawning player {player:?} entity for game");
+                        info!("Spawning player in server hosted  game {player:?}");
```
Comment (cBournhonesque) [inline examples/lobby/src/server.rs:245]:
```suggestion
                        info!("Spawning player {player:?} in server hosted game");
```

File: examples/lobby/src/server.rs [hunk 2]
```diff
@@ -241,11 +237,12 @@ mod lobby {
                         host: lobby.host,
                     },
                 );
-            } else {
+            } // Common scenario player just started and everyone is in lobby
```
Comment (cBournhonesque) [inline examples/lobby/src/server.rs:240]: I would remove this comment

**Comment by cBournhonesque** [comment]
Thanks, I left a couple comments!

File: examples/lobby/src/server.rs (hunk 2 above)
Comment (Sirmadeira) [reply examples/lobby/src/server.rs]: Fixed

File: examples/lobby/src/server.rs (hunk 1 above)
Comment (Sirmadeira) [reply examples/lobby/src/server.rs]: Fixed

**Comment by Sirmadeira** [comment]
Can I make it UI based instead o egui?

**Comment by cBournhonesque** [comment]
> Can I make it UI based instead o egui? More newbee friendly, also dissociate a few modules

Sure, if you manage to make it work!

**Pull request merged**